packages/composites/proof-of-reserves/src/utils/protocol.ts

```diff
@@ -19,6 +19,7 @@ import { adapter as staderList } from '@chainlink/stader-address-list-adapter'
 import * as swellList from '@chainlink/swell-address-list-adapter'
 import { adapter as wBTC } from '@chainlink/wbtc-address-set-adapter'
 import * as wrapped from '@chainlink/wrapped-adapter'
+import { adapter as coinbasePrime } from '@chainlink/coinbase-prime-adapter'
```

Comment:
I think the coinbase-price needs to be added to the references of tsconfig of proof-of-reserves EA so that the `build` works 

`{ "path": "../../sources/coinbase-prime" }`

Reply:
added